src/mower_logic/src/mower_logic/behaviors/AreaRecordingBehavior.cpp

```diff
@@ -269,7 +273,18 @@ void AreaRecordingBehavior::joy_received(const sensor_msgs::Joy &joy_msg) {
         set_docking_position = true;
     }
 
-
+    // use RB button for manual point collecting
+    // enable/disable auto point collecting with LB+RB
+    if (joy_msg.buttons[5] && !last_joy.buttons[5]) {
+        if (joy_msg.buttons[4] && !last_joy.buttons[4]) {
```

Comment:
this if will likely never happen as there is no separate state for joy buttons, so all four conditions will be not met (?)

Reply:
yes, that would require pressing both buttons in the same loop